test/contacts.js

```diff
@@ -10,228 +10,231 @@ client.useKey(api_key);
-            client.contacts.getByEmailBatch({
-                emails: ['testingapis@hubspot.com', 'testingapisawesomeandstuff@hubspot.com']
-            },function(err, data, res) {
-                if (err) { throw err; }
-                expect(res.statusCode).to.equal(200);
-                expect(res).to.be.defined;
-                done();
-            })
-        });
+  describe('Get A Contact By Email Batch', function(){
+    it('Should return a contact record based on a array of emails', function (done) {
+      client.contacts.getByEmailBatch({
+        emails: ['testingapis@hubspot.com', 'testingapisawesomeandstuff@hubspot.com']
+      },function(err, data, res) {
+        if (err) { throw err; }
+        expect(res.statusCode).to.equal(200);
+        expect(res).to.be.a('object');
+        done();
+      })
     });
+  });
 
-    describe('Update A Contact', function(){
-        it('Should update an existing contact', function (done) {
-            client.contacts.update(61571, {
-                "properties": [
-                {
-                    "property": "email",
-                    "value": "new-email@hubspot.com"
-                },
-                {
-                    "property": "firstname",
-                    "value": "Updated",
-                    "timestamp": 1419284759000
-                },
-                {
-                    "property": "lastname",
-                    "value": "Lead",
-                    "timestamp": 1419284759000
-                },
-                {
-                    "property": "website",
-                    "value": "http://hubspot-updated-lead.com"
-                },
-                {
-                    "property": "lifecyclestage",
-                    "value": "customer"
-                }
-            ]
-        },function(err, data, res) {
-                if (err) { throw err; }
-                expect(data).to.be.defined;
-                expect(data.properties).to.be.defined;
-                done();
-            })
-        });
+  describe('Update A Contact', function(){
+    it('Should update an existing contact', function (done) {
+      client.contacts.update(61571, {
+        "properties": [
+          {
+            "property": "email",
+            "value": "new-email@hubspot.com"
+          },
+          {
+            "property": "firstname",
+            "value": "Updated",
+            "timestamp": 1419284759000
+          },
+          {
+            "property": "lastname",
+            "value": "Lead",
+            "timestamp": 1419284759000
+          },
+          {
+            "property": "website",
+            "value": "http://hubspot-updated-lead.com"
+          },
+          {
+            "property": "lifecyclestage",
+            "value": "customer"
+          }
+        ]
+      },function(err, data, res) {
+        if (err) { throw err; }
+        expect(res.statusCode).to.equal(404);
+        expect(data.status).to.equal('error');
```

Comment:
Not sure how this slipped my view but will change it to actually have usefull errors.